app/Models/Signup.php

```diff
@@ -22,7 +22,7 @@ class Signup extends Model
      * @var array
      */
     public static $indexes = [
-        'campaign_id', 'campaign_run_id',
+        'campaign_id', 'campaign_run_id', 'updated_at',
```

Comment:
@katiecrane that makes good sense to me - I just didn't know that before so thanks for letting me know! I added the migration :)